website/source/docs/providers/aws/r/emr_cluster.html.md

```diff
@@ -71,13 +71,13 @@ The following arguments are supported:
 * `core_instance_count` - (Optional) Number of Amazon EC2 instances used to execute the job flow. EMR will use one node as the cluster's master node and use the remainder of the nodes (`core_instance_count`-1) as core nodes. Default `1`
 * `log_uri` - (Optional) S3 bucket to write the log files of the job flow. If a value
 	is not provided, logs are not created
-* `applications` - (Optional) A list of applications for the cluster. Valid values are: `Flink`, `Hadoop`, `Hive`, `Mahout`, `Pig`, and `Spark`. Case insensitive
+* `applications` - (Optional) A list of applications for the cluster. Values are passed directly to the AWS API, valid values varies with EMR release, examples include: `Flink`, `Ganglia`, `Hadoop`, `Hive`, `Mahout`, `Pig`, and `Spark`. Case insensitive
```

Comment:
I reckon this list will keep growing over time and we'd be playing 😼  & 🐭  game here with Amazon. What do you think about linking to the official docs that would provide the list + mentioning just a few examples (not an extensive list)?

e.g. http://docs.aws.amazon.com//emr/latest/ReleaseGuide/emr-release-components.html